From ab79fb3f57f7009882bd1d27f5f5505e46385579 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Thu, 19 Mar 2020 00:48:07 +0200
Subject: [PATCH] Revert "[SW-0] goya2: ppboot: set sp/fp at crt0"

This reverts commit ede2b8a6bb102e06a96d459fbd1edfd258c7f6d2.
arcboot_rom calls ppboot as a function and expects it to return
hence sharing same stack on DCCM

Change-Id: I648f25265e9624d1d6160ae6066904a23c26676e
---

diff --git a/app/crt0_ppboot.S b/app/crt0_ppboot.S
index c8d0d69..b4bd0bb 100644
--- a/app/crt0_ppboot.S
+++ b/app/crt0_ppboot.S
@@ -16,9 +16,6 @@
 /* lf2flt: to avoid compiler (buildroot wrapper) warning during linkage */
 .global	lf2flt
 lf2flt:
-
-	mov	sp, __stack_top
-	mov	fp, sp
 #endif
 
 #ifdef __aarch64__
